src/main/java/com/backbase/ct/bbfuel/client/payment/PaymentOrderPresentationRestClient.java

```diff
@@ -16,14 +16,15 @@
     private final BbFuelConfiguration config;
 
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```

Comment:
I suggest to call this constant `CLIENT_API`

Reply:
done